tests/unittest/llmapi/apps/openai_server.py

```
@@ -105,26 +105,32 @@ def _wait_for_server(self, *, url: str, timeout: float):
         # run health check on the first rank only.
         start = time.time()
         while True:
+            err = None
             try:
                 if self.rank == 0:
                     if requests.get(url).status_code == 200:
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)